package.json

```
@@ -20,7 +20,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint ."
```

Hey - this looks weird why not use....
```suggestion
    "lint": "eslint abcde ."
```